db_sql.go

```diff
@@ -179,7 +177,7 @@ func (x *sqlUserStoreDB) CreateIdentity(email, username, firstname, lastname, mo
 			}
 		}
 
-		if x.enableAuthenticator {
+		if authUserType == IMQSUserType {
 			// Insert into auth user
```

Comment:
This comment is a lie - there is no "auth user" table - just remove the comment.